src/Admins/SearchAdmin.php

```diff
@@ -2,23 +2,20 @@
 
 namespace Firesphere\SolrSearch\Admins;
 
-use Firesphere\SolrSearch\Models\SearchClass;
-use SilverStripe\Admin\ModelAdmin;
-
 /**
  * Class \Firesphere\SolrSearch\Admins\SearchAdmin
  * @todo implement search administration, e.g. Elevation and Facets
  *
-class SearchAdmin extends ModelAdmin
-{
-    private static $managed_models = [
-        SearchClass::class
-    ];
-
-    private static $menu_icon_class = 'font-icon-search';
-
-    private static $url_segment = 'searchadmin';
-
-    private static $menu_title = 'Search';
-}
-/** **/
+ * class SearchAdmin extends ModelAdmin
```

Comment:
Commented out because it has no value yet at this point. The reason for this Admin to exist, came from the original thought of using it to manage indexed fields, which is pushed down the backlog currently.

It can be re-activated for other uses (e.g. logging/elevation etc.)